src/SDKs/SignalR/Management.SignalR/Properties/AssemblyInfo.cs

```diff
@@ -8,8 +8,8 @@
 [assembly: AssemblyTitle("Microsoft Azure SignalR Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure SignalR Resources.")]
 
-[assembly: AssemblyVersion("0.10.0.0")]
-[assembly: AssemblyFileVersion("0.10.0.0")]
+[assembly: AssemblyVersion("0.10.1.0")]
```

Comment:
@maddieclayton update this version to 0.11.0 (both assemblyversion as well assemblyfile version)